client/src/backoffice/EventsPage.tsx

```diff
@@ -117,49 +154,130 @@ const EventForm: React.FC<{ create?: boolean; event?: PoapEvent }> = ({ create,
                 <EventField disabled={!create} title="ID" name="id" />
               </>
             )}
-
-            <EventField disabled={!create} title="Fancy ID" name="fancy_id" />
-            <EventField disabled={!create} title="Description" name="description" />
-            <div className="bk-group">
-              <EventField disabled={!create} title="Start Date" name="start_date" />
-              <EventField disabled={!create} title="End Date" name="end_date" />
-            </div>
+            <EventField disabled={!create} title="Description" type="textarea" name="description" />
             <div className="bk-group">
               <EventField disabled={!create} title="City" name="city" />
               <EventField disabled={!create} title="Country" name="country" />
             </div>
+            <div className="bk-group">
+              <DayPickerContainer
+                text="Start Date:"
+                dayToSetup="start_date"
+                handleDayClick={handleDayClick}
+                setFieldValue={setFieldValue}
+                disabledDays={
+                  values.end_date ? { from: new Date(values.end_date), to: endingDate } : undefined
+                }
+              />
+              <DayPickerContainer
+                text="End Date:"
+                dayToSetup="end_date"
+                handleDayClick={handleDayClick}
+                setFieldValue={setFieldValue}
+                disabledDays={
+                  values.start_date
+                    ? { from: startingDate, to: new Date(values.start_date) }
+                    : undefined
+                }
+              />
+            </div>
             <EventField title="Website" name="event_url" />
-            <EventField title="Image Url" name="image_url" />
-            <EventField title="Signer Url" name="signer_ip" />
-            <EventField title="Signer Address" name="signer" />
 
-            <SubmitButton text="Save" isSubmitting={isSubmitting} canSubmit={dirty && isValid} />
+            <ImageContainer
+              text="Upload Image:"
+              handleFileChange={handleFileChange}
+              setFieldValue={setFieldValue}
+              errors={errors}
+            />
+            {event && typeof event.image === 'string' && (
+              <div className={'image-edit-container'}>
+                <img className={'image-edit'} src={event.image} />
+              </div>
+            )}
+            <SubmitButton text="Save" isSubmitting={isSubmitting} canSubmit={true} />
           </Form>
         )}
       </Formik>
-      {!create && <Link to={`/claim/${event!.fancy_id}`}>Go to Claim Page</Link>}
     </div>
   );
 };
 
+type DatePickerContainerProps = {
+  text: string;
+  dayToSetup: 'start_date' | 'end_date';
+  handleDayClick: Function;
+  setFieldValue: Function;
```

Comment:
Functions are defined with its parameters types and outputs types.

Code example:
```typescript
handleDayClick: (day: date) => void;
setFieldValue: () => void;
```